src/main/webapp/scripts/ou-script.js

```diff
@@ -59,29 +134,50 @@ function getAllOUs() {
 }
 
 /*
- * Computes an org unit's depth using the number of slashes in its path.
-*/
-function computeDepth(orgUnit) {
-    // +1, as the root OU is considered layer 1
-    return (orgUnit['orgUnitPath'].match(/\//g) || []).length + 1;
+ * Constructs the no matching search results screen.
+ */
+function noSearchResult() {
+    if (document.getElementById("no-search-result-elem")) {
+        isLoading = false;
+        setLoadingOverlay();
+        return;
+    }
+    var div = document.createElement("div");
+        div.classList.add("no-search-results");
```

Comment:
these lines shouldn't be indented

Reply:
variables, div,p, and btn should all be renamed to be more descriptive.

Reply:
fixed